modules/server/auth/auth.service.tmpl

```diff
@@ -0,0 +1,34 @@
+[Unit]
+Description=Auth
+After=docker.service,haveged.service,searchbroker.service
+Requires=docker.service
+
+[Service]
+EnvironmentFile=/mnt/secrets/auth
+ExecStartPre=-/usr/bin/docker kill auth
+ExecStartPre=-/usr/bin/docker rm auth
+ExecStartPre=/usr/bin/docker pull martinbreu/auth:${version}
+ExecStart=/usr/bin/docker run \
+  --name=auth \
+  --network=searchbroker \
+  -e TOMCAT_USERNAME='admin' \
+  -e TOMCAT_PASSWORD='$${TOMCAT_PASS}' \
+  -e POSTGRES_HOST='postgres.openstacklocal' \
+  -e POSTGRES_DB='auth_db' \
+  -e POSTGRES_PORT='5432' \
```

Comment:
We should not have such defaults here.

Reply:
With such you mean POSTGRES_PORT? Should I store the postgres settings elsewhere and access from MDR, Searchbroker and Auth?

Reply:
Config of the default Postgres Port 5432 should not be needed here.